CMakeLists.txt

```diff
@@ -37,18 +37,20 @@ option(USE_CMAKE_INSTALL
   "Install build products using cmake's install() instead of the bootstrap script's install()"
   ${CMAKE_INSTALL_DEFAULT})
 
-find_package(TSC CONFIG REQUIRED)
+if(BOOTSTRAP)
```

Comment:
Please add a proper `option` if you are adding a new option above to provide documentation of what it does and to make it possible to identify the need for the value when directly using CMake.

Reply:
Will do.